contracts/src/v0.8/ccip/test/pools/TokenPool/TokenPool.setChainRateLimiterConfigs.t.sol

```diff
@@ -51,20 +80,34 @@ contract TokenPool_setChainRateLimiterConfig is TokenPoolSetup {
   // Reverts
 
   function test_OnlyOwnerOrRateLimitAdmin_Revert() public {
+    uint64[] memory chainSelectors = new uint64[](1);
+    chainSelectors[0] = DEST_CHAIN_SELECTOR;
+
+    RateLimiter.Config[] memory newOutboundConfigs = new RateLimiter.Config[](1);
```

Comment:
I prefer the absolute minimal setup for reverts, meaning we can pass in empty lists here and remove most of this setup